src/test/java/com/wikia/webdriver/testcases/mercurytests/HeaderTest.java

```diff
@@ -0,0 +1,99 @@
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.Header;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class HeaderTest extends NewTestTemplate {
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithoutInfobox")
+  public void mercury_header_checkElementsVisibilityWithoutInfobox() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.NO_INFOBOX)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is visible");
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    Assertion.assertTrue(header.isHeroImageVisible(), "hero image is visible");
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_2)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is visible");
+  }
+
+  @Test(groups = "mercury_header_heroImageIsProperlyStyled")
+  public void mercury_header_heroImageIsProperlyStyled() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    Assertion.assertEquals(
+        header.getHeroImageCssValue("background-color"),
+        "rgba(255, 255, 255, 1)",
+        "hero image has proper background-color attribute"
+    );
+    Assertion.assertEquals(
+        header.getHeroImageCssValue("background-position"),
+        "50% 50%",
+        "hero image has proper background-position attribute"
+    );
+    Assertion.assertEquals(
+        header.getHeroImageCssValue("background-repeat"),
+        "no-repeat",
+        "hero image has proper background-repeat attribute"
+    );
+  }
+
+  @Test(groups = "mercury_header_heroImageIsSquare")
+  public void mercury_header_heroImageIsSquare() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)